en/docs/install-and-setup/setup/kubernetes-operators/k8s-api-operator/manage-integration/integration-deployments.md

```diff
@@ -450,4 +454,170 @@ Use the following methods to invoke the inbound endpoints in HTTP and HTTPS tran
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: integration-config
+data:
+  requestCPU: "500m"
+  reqMemory: "512Mi"
+  cpuLimit: "2000m"
+  memoryLimit: "2048Mi"
+```
+
+Please refer to [Managing Resources for Containers](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) to learn more. 
+
+
+## Enable Auto Scaling for deployment
+
+When traffic is increased your pods may need to horizontally scaled. Kubernetes provides a way to define resource limits and a policy for the deployment so that it will auto scale the deployment looking at resource usage. Please read on [Horizontal Pod Scaling (HPA)](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/) in Kubernetes.
+
+You can enable scaling and define maximum number of replicas to scale under `autoScale` of `integration_cr.yaml` as below. 
+
+  ```yaml
+apiVersion: wso2.com/v1alpha2
+kind: Integration
+metadata:
+  name: test-integration
+spec:
+  autoScale:
+    enabled: "true"
+    maxReplicas: 3
+  ```
+
+You can enable auto scaling by default by defining it at `integration_controller_conf.yaml` file. These values will get overridden by what is defined at  `integration_cr.yaml` file. 
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: integration-config
+data:
+  enableAutoScale: "true"
+  maxReplicas: "5"
+```
+
+Note how you can set required resources and resource limits for the pods in the deployment referring to the above section [Managing resources for pods in EI deployment](#Managing-resources-for-pods-in-EI-deployment). HPA configs are injected through the integration configmap. See how you can define it at [integration_controller_conf.yaml]([integration_controller_conf.yaml](https://github.com/wso2/k8s-api-operator/blob/master/api-operator/deploy/controller-configs/integration_controller_conf.yaml)) file. 
+
+```yaml
+  hpaMetrics: |
+    - type: Resource
+      resource:
+        name: cpu
+        target:
+          type: Utilization
+          averageUtilization: 50
+```
+
+## Liveness and readiness probes 
+
+The operator gives you the flexibility to define liveness and readiness probes. These probes are used by Kubernetes to know whether pods are live and whether they are ready to accept traffic. 
+
+You can define the default probe definitions at `integration_controller_conf.yaml` file and include them to the configMap. Usually, WSO2 micro integrator listens for HTTP traffic on 8290 port. Liveness of the pod is checked by a ping to that port. WSO2 micro integrator is ready to accept traffic when all the CApps are successfully deployed. The API with path `/healthz` which is exposed through port 9201 is used for that. Note that these ports can change as per the configurations used at the MI based image. 
```

Comment:
Shall we change this phase to "listens for HTTP traffic on passthrough port(default 8290)"?

Reply:
Also, "HTTP inbound port(default 9201)" instead 9201

Reply:
+1. Updated.